### PR TITLE
Fix minor typos in Docstrings

### PR DESCRIPTION
**Pull request by abhi-glitchhg**

### CODE REVIEW

**Review by datumbox — APPROVED** [review]
LGTM, thanks!

File: torchvision/datasets/food101.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
 class Food101(VisionDataset):
     """`The Food-101 Data Set <https://data.vision.ee.ethz.ch/cvl/datasets_extra/food-101/>`_.
 
-    The Food-101 is a challenging data set of 101 food categories, with 101'000 images.
+    The Food-101 is a challenging data set of 101 food categories, with 101,000 images.
```
Comment (NicolasHug) [inline torchvision/datasets/food101.py:14]: The double comma might make it a bit hard to parse, might be worth dropping one of them

```suggestion
    The Food-101 is a challenging data set of 101 food categories with 101,000 images.
```

**Review by NicolasHug — APPROVED** [review]
Minor comment but LGTM anyway, thanks @abhi-glitchhg
